jekyll/_cci2/release/configure-release-markers.adoc

```diff
@@ -0,0 +1,38 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure release markers
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure release markers in your workflow
+:icons: font
+:experimental:
+
+This tutorial shows how to add a "release marker" step to your workflow config. Release markers allow you log all new releases in one place and link back to the CI/CD pipelines that caused them, without needing to install the release agent. 
+
+[#prerequisites]
+== Prerequisites
+
+Before adding a release marker step to your workflow, you will need to create a new environment integration in the CircleCI Releases UI. To do this, you can follow steps 1-4 from the xref:set-up-circleci-releases#set-up-releases[Set up CircleCI releases] page. You don't need to install a release agent, but you will need to reference this environment integration name as part of your config when adding the `log release` step below.
```

Comment:
the instructions we link to say make env k8s cluster. but we should probably mention using custom? or make instructions more general? or just dupe and mod instructions locally

Reply:
I've updated it to include the steps on this page.